### PR TITLE
made bubble colors more accessible

### LINKED ISSUE #4: Change the red bubble color to a more accessible and legible color.

### PR DESCRIPTION
**Pull request by kreisdavid**
### What does this PR do?
- Changes discipline bubble colors (which previously were red and did not meet web content accessibility guidelines) to a color palette that was more accessible. 

### What issues does it address?
- Closes #4 

### How to test:
- Access staging server (see slack for details)
- Go to arc-visualization directory 
`[bigdiva@bigdiva-staging ~]$ cd arc-visualization/`
- Run `$ rvmsudo unicorn_rails -o 192.237.180.236 -p 80`
- Access the BigDiva page by going to 192.237.180.236
- Notice the circles next to "Visualize By:" under the BigDiva logo.
- Note that there is no red circle, rather a vivid amber to the right of "Discipline".
- Click on "Discipline" and note that the bubbles that appear are not red, but the same vivid amber.
- To compare what the red used to look like, go to http://bigdiva.org/

### CODE REVIEW

File: app/assets/javascripts/home.js [hunk 1]
```diff
@@ -1028,10 +1028,10 @@ $(function() {
       {"id":"gradient-genre-collapsed",  "color":"#b2e2e2", "highlight":"#006d2c"},
       {"id":"gradient-genre-fixed",      "color":"#006d2c", "highlight":"#66c2a4"},
       {"id":"gradient-genre-selected",   "color":"#2ca25f", "highlight":"#edf8fb"},
-      {"id":"gradient-discipline-normal",   "color":"#b30000", "highlight":"#e34a33"}, // reds: #fef0d9, #fdcc8a, #fc8d59, #e34a33, #b30000
-      {"id":"gradient-discipline-collapsed","color":"#fdcc8a", "highlight":"#b30000"},
-      {"id":"gradient-discipline-fixed",    "color":"#b30000", "highlight":"#fc8d59"},
-      {"id":"gradient-discipline-selected", "color":"#e34a33", "highlight":"#fef0d9"},
+      {"id":"gradient-discipline-normal",    "color":"#e2aa00", "highlight":"#efb915"}, // NOW: yellows: #fff6db, #ffdd82, #ffce3d, #efb915, #e2aa00
+      {"id":"gradient-disipline-collapsed",  "color":"#ffdd82", "highlight":"#e2aa00"},
```
Comment (jamiefolsom) [inline app/assets/javascripts/home.js]: Is this a typo, or is there a reason you changed the spelling of the selector?

**Review by jamiefolsom — COMMENTED** [review]
One nit about the spelling of a CSS selector. Once we get that resolved, and we get clarity on the staging server for testing, I think we'll be good to go. You can remove WIP from the PR title, and I will ping Nick in the AM about staging. Nice work!

File: app/assets/javascripts/home.js (hunk 1 above)
Comment (kreisdavid) [reply app/assets/javascripts/home.js]: That would be a typo, good catch.

**Review by jamiefolsom — APPROVED** [review]
Looks good 👍 once I've set up the staging server, you can deploy your branch there for testing, request a test from Steve, and if he signs off, merge to master.

**Comment by SteveMarvin** [comment]
Looks good! I love these kinds of visualizations. ;-) Accepted 👍

**Pull request merged**